nss-day-cohort-52/giffygram-flying-racoons
# Added CSS, DM form, navbar

#### Changes Made
1. added a navbar module with html for the navbar 
2. added html for the message form in the messageForm module
3. added POST function to messageForm module to post the message to the database

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout ad-messages
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > serve the branch. The page should look like the original website with header and footer and post list formatting matching. You should be able to click the pen icon to display the message form. the send, cancel, and close buttons should all be functional. after clicking send, you should get a window alert telling you the message sent and the message should show up in the database.
    [ ] Does app function with change?
    
4. View code file.
    [ ] Confirm file modifications are present as indicated above.
    [ ] Confirm no unused code or extraneous comments exist.
